package.json

```diff
@@ -132,6 +132,7 @@
     "@types/getos": "^3.0.4",
     "@types/js-yaml": "^4.0.9",
     "@types/node": "^20",
+    "@types/semver": "^7.5.8",
```

Comment:
The dependency `semver` and `@types/semver` should probably be added in `extensions/kubectl-cli/package.json`